pkg/registry/generic/registry/store.go

```diff
@@ -40,6 +41,10 @@ import (
 	"github.com/golang/glog"
 )
 
+// EnableGarbageCollector affects the handling of Update and Delete requests. It
+// must be synced with the corresponding flag in kube-controller-manager.
+var EnableGarbageCollector bool
```

Comment:
@liggitt my only complain about RESTOptions is that it's populated to the storage in individual `NewREST()` functions (e.g. https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/podsecuritypolicy/etcd/etcd.go#L59). If someone creates a new registry and forgets to populate the option, it will cause hard to find bugs.